custom_components/midea_ac_lan/config_flow.py

```diff
@@ -196,7 +192,8 @@ async def async_step_login(
                 )
                 if self.cloud is None:
                     raise AbortFlow(
-                        f"Can not get midea cloud: {SERVERS[user_input[CONF_SERVER]]}"
+                        f"Can not get midea cloud: {
+                            SERVERS[user_input[CONF_SERVER]]}",
```

Comment:
Python version 3.11 does not allow new lines in expression parts of non-triple-quoted f-strings